workflows/dragen-somatic-with-germline-pipeline/4.3.6/dragen-somatic-with-germline-pipeline__4.3.6.cwl

```diff
@@ -918,6 +918,23 @@ inputs:
       - pattern: ".tbi"
         required: true
 
+  # Cross sample contamination
+  qc_cross_cont_vcf:
+    label: qc cross cont vcf
+    doc: |
+      The cross-contamination metric is enabled by including one of the following flags along with a compatible VCF.
+      Pre-built contamination VCF files for different human references can be found at /opt/edico/config. 
+      DRAGEN supports separate modes for germline and somatic samples. 
+    type: File?
+
+  qc_somatic_contam_vcf:
+    label: qc somatic contam vcf
+    doc: |
+      The cross-contamination metric is enabled by including one of the following flags along with a compatible VCF.
+      Pre-built contamination VCF files for different human references can be found at /opt/edico/config. 
+      DRAGEN supports separate modes for germline and somatic samples. 
+    type: File?
```

Comment:
Enum option still needs to be provided here

Reply:
Same as above.